av1an-core/src/broker.rs

```diff
@@ -193,6 +196,11 @@ impl<'a> Broker<'a> {
           // avoids double-print of the error message as both a WARN and ERROR,
           // since `Broker::encoding_loop` will print the error message as well
           warn!("Encoder failed (on chunk {}):\n{}", chunk.index, e);
+
+          if format!("{:?}", e.stderr).contains("av1_tpl_stats_ready") {
```

Comment:
Can we do this without an allocation and UTF-8 checking? It looks like we already have `memchr` in our dependency tree transitively, so we could use something like [`memmem`](https://docs.rs/memchr/2.4.1/memchr/memmem/index.html) to just search for a byte slice directly, without having to do another allocation or verify the UTF-8.

Reply:
I'd actually like to figure out how to do that, if someone has worked with av1an's `StringOrBytes` type before? Because that is what we are working with here. Is the thing to do just a `match` and test both branches?

Reply:
Yeah actually now that you mention that the type is `StringOrBytes`, this wouldn't actually work for the `Bytes` case since "Aa" would be formatted as `"raw bytes: [65, 97]"` (since it's using debug formatting, see `Debug` impl for `StringOrBytes`), so you can't actually check the string directly. But I think pattern matching and using `.contains()` would be fine in the string case, and using `memmem` in the bytes case.

Reply:
Actually, we can just always use `memmem` since it should work for the regular string types also. You'd just have to call `.as_bytes()` on the string. That's probably the simplest way to do it.

Reply:
Actually, I think the absolute simplest and cleanest way to do it would to just add an `as_bytes` function in the impl of `StringOrBytes` itself so that there is no code duplication in the future either. Then we'd just call newly-added `as_bytes` method and pass the bytes to `memmem`.